Dockerfile

```diff
@@ -1,8 +1,13 @@
 FROM jupyter/datascience-notebook
 
+LABEL org.opencontainers.image.source https://github.com/cal-itp/calitp-py
+
 USER root
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
 RUN apt update \
-    && apt install keychain
+    && apt install keychain \
+    && apt-get update \
```

Comment:
I think the second update should be unnecessary.

I would also just do `apt install -y nodejs`